features/working_with_legacy_code/message_chains.feature

```diff
@@ -1,6 +1,7 @@
 Feature: Message Chains
 
-  You can use `receive_message_chain` in place of `receive` to stub a chain of messages:
+  You can use `receive_message_chain` in place of `receive` in certain instances
```

Comment:
"in certain instances" reads a little funny to me, in certain circumstances would probably be better but I'm not sure this change is needed at all